RadixWallet.xcodeproj/project.pbxproj

```diff
@@ -7850,6 +7887,28 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
```

Comment:
Do we still want this?

Reply:
Yes, otherwise developers outside RDX Works team won't have the placeholder `SensitiveInfo.plist` created (as they won't run fastlane).

Reply:
why wouldn't they run the fastlane?

Reply:
well, until now fastlane only makes sense if you have access to secrets. We can remove this script phase and update development set up so that they always run the `create_sensitive_info` lane though. wdyt?

Reply:
well, the respective lane that you have added those not require secrets, unless I am missing something. Those are needed when the app has to be built.